src/main/java/it/pn/frontend/e2e/pages/radd/StoricoRichiestePage.java

```diff
@@ -57,6 +57,9 @@ public class StoricoRichiestePage extends BasePage {
     @FindBy(xpath = "//input[contains(@value,'AOR')]")
     WebElement AARRadioButton;
 
+    @FindBy(xpath = "//input[contains(@value,'PG')]")
```

Comment:
inserire id

Reply:
for this element there is no Id: input class="PrivateSwitchBase-input css-1m9pwf3" name="recipientType" type="radio" value="PG"